selfdrive/test/helpers.py

```diff
@@ -79,3 +80,11 @@ def read_segment_list(segment_list_path):
     seg_list = f.read().splitlines()
 
   return [(platform[2:], segment) for platform, segment in zip(seg_list[::2], seg_list[1::2], strict=True)]
+
+
+# Utilities for sanitizing routes of only essential data for testing car ports and doing validation.
+
+PRESERVE_SERVICES = ["can", "carParams", "pandaStates", "pandaStateDEPRECATED"]
+
+def sanitize(lr: LogIterable) -> LogIterable:
+  return filter(lambda msg: msg.which() in PRESERVE_SERVICES, lr)
```

Comment:
Always two spaces around top level functions.

Reply:
Also, you probably want to zero out the carVin inside carParams

Reply:
maybe just the last few digits (the serial number)? we might want to use the vin for fingerprinting on some brands

Reply:
that's fine, I'm hiding the last 8 on the FP bot